.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+
+jobs:
+  ruby_rails_test_matrix:
```

```suggestion
  test:
```
nit, only because there's no rails in this job